crates/runtime/src/embeddings/udtf.rs

```diff
@@ -464,9 +465,22 @@ impl TableProvider for VectorSearchUDTFProvider {
                 false,
             )],
             input: Arc::new(proj),
-            fetch: limit,
+            // i.e., the order limit passed down via the UDTF invocation
+            fetch: self.args.limit,
         });
 
-        state.create_physical_plan(&sort).await
+        /* TODO: for a query such as
```

Comment:
Can you make a tracking issue and reference it here?

Reply:
#6639 + linked in source